app/services/openid_connect_redirector.rb

```diff
@@ -15,10 +15,15 @@ def initialize(redirect_uri:, service_provider:, state:, errors: nil, error_attr
     @redirect_uri = redirect_uri
     @service_provider = service_provider
     @state = state
-    @errors = errors
+    @errors = errors || ActiveModel::Errors.new(self)
```

Comment:
CodeClimate is complaining here but I don't think there's a straightforward solution (especially since we need the reference to `self`...which we can't get until after the object is initialized....) so I think we should just add this to the `.reek` to ignore. Sorry!